utils/TokenData.ts

```diff
@@ -2,6 +2,7 @@ import { defaults } from 'lodash';
 
 export const DEFAULT_DECIMALS = 18;
 export const NATIVE_TOKEN_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
+export const WETH_ADDRESS = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2';
```

Comment:
We should only add symbolic/chain-agnostic data here, so `WETH_ADDRESS` should be either fetched from `data.ts` or used defined as a mock address in `test/bancor-portal/BancorPortal.ts` instead

Reply:
done